File: packages/durable-streams/packages/client-java/src/main/java/com/durablestreams/exception/StreamExistsException.java
package com.durablestreams.exception;

/**
 * Thrown when trying to create a stream that already exists (409).
 */
public class StreamExistsException extends DurableStreamException {
    public StreamExistsException(String url) {
        super("Stream already exists: " + url, 409, "CONFLICT");
    }
}

File: packages/durable-streams/packages/client-php/src/Exception/StreamExistsException.php
<?php

declare(strict_types=1);

namespace DurableStreams\Exception;

class StreamExistsException extends DurableStreamException
{
    public function __construct(string $url, ?\Throwable $previous = null)
    {
        parent::__construct(
            "Stream already exists: {$url}",
            'CONFLICT_EXISTS',
            409,
            [],
            $previous
        );
    }
}
